src/Validators/BaseValidator.php

```diff
@@ -111,8 +111,11 @@ public static function validateId($field, $table, $lookupId, $isUuid = false)
             "SELECT $field FROM $table WHERE $field = ?",
             array($lookupId)
         );
-
-        return $result[$field] ? true : $validationResult;
+        if ($result) {
```

Comment:
guessing this had to be changed for php8 issues. would change this if to:
```
if (!empty($result[$field])) {
```

Reply:
Done